contrib/examples/example_add_bot/setup.py

```diff
@@ -0,0 +1,39 @@
+from setuptools import find_packages, setup
+
+setup(
+    name='intelmqExampleAddBot',
+    version=3.1,  # noqa: F821
```

Comment:
should be a string AFAIK. And let's use semantic versioning
```suggestion
    version='3.1.0',
```